.github/pull_request_template.md

```diff
@@ -0,0 +1,75 @@
+⚠️ Click preview button above ⚠️
+
+# PR Template
+
+1. Apply relevant labels.
+1. Self-review code and PR.
+1. Request review from a team member.
+1. Wait for the code review.
+1. Take the bits that apply to your PR and delete the rest of template.
+
+---
+
+## Solution
+
+- Describe **how** your solution works and solves the problem. Speak to your design and the approach taken in the implementation.
+- Call out any assumptions and key decisions made while solving the issue.
+- Call out changes that may lead to deployment failure
+  - new or updated test specs
+  - dependency and other package updates (use label and link to dependency docs)
+  - new migrations (use label)
+  - devOps changes (use label)
+
+```md
+###### Thing your're calling out
+
+The thing being called out. Use a list when it makes sense.
```

Comment:
What thing is being called out here? Can you elaborate on this?

Reply:
It's up to the author, I can't say here.